tools/install/package.json

```diff
@@ -18,10 +19,12 @@
     "@lg-tools/meta": "workspace:^",
     "@types/cross-spawn": "6.0.2",
     "cross-spawn": "7.0.3",
-    "node-fetch": "2.6.12"
+    "chalk": "4.1.2"
   },
   "devDependencies": {
-    "@types/node-fetch": "^2.6.11"
+    "@lg-tools/lint": "workspace:^",
+    "@types/node-fetch": "^2.6.11",
```

Comment:
Do you still need this since you removed `node-fetch`?

Reply:
Nope, good catch. Not sure how that snuck back in